<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>487</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>717</width>
    <height>487</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reminder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qrcrem.qrc">
    <normaloff>:/images/reminder.ico</normaloff>:/images/reminder.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth">
           <number>7</number>
          </property>
          <widget class="QTextEdit" name="textEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>520</width>
             <height>60</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>526</width>
             <height>203</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
          <widget class="QTextEdit" name="textEditOut">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>520</width>
             <height>50</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>526</width>
             <height>202</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="BrightText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>226</red>
                 <green>226</green>
                 <blue>226</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="BrightText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>226</red>
                 <green>226</green>
                 <blue>226</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="BrightText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>240</green>
                 <blue>240</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pBAdd">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Сохранить напоминание</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>10</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QDateEdit" name="dateEdit">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="minimumDate">
               <date>
                <year>2018</year>
                <month>1</month>
                <day>1</day>
               </date>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
              <property name="date">
               <date>
                <year>2018</year>
                <month>1</month>
                <day>1</day>
               </date>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="displayFormat">
               <string>HH:mm</string>
              </property>
              <property name="calendarPopup">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>10</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBList">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Список напоминаний</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBHistList">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>История напоминаний</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>10</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBAddFiles">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Добавить файлы...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBAddDirs">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Добавить каталоги...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pBDonate">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Отблагодарить автора</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lbClock">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actHide"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actAbout"/>
    <addaction name="separator"/>
    <addaction name="actionReadme_txt"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Звуки</string>
     </property>
     <addaction name="actNoSound"/>
     <addaction name="separator"/>
     <addaction name="actionbushorn"/>
     <addaction name="actioncow"/>
     <addaction name="actionhey"/>
     <addaction name="actionkostel"/>
     <addaction name="actionphone"/>
     <addaction name="separator"/>
     <addaction name="actMySound"/>
    </widget>
    <addaction name="actCancelFiles"/>
    <addaction name="actCancelDirs"/>
    <addaction name="separator"/>
    <addaction name="actAutoOpenFiles"/>
    <addaction name="actAutoOpenDirs"/>
    <addaction name="separator"/>
    <addaction name="menu_4"/>
    <addaction name="separator"/>
    <addaction name="actAutoLoad"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actExit">
   <property name="text">
    <string>&amp;Выход (Esc)</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actHide">
   <property name="text">
    <string>Свернуть в трей (Ctrl+Q)</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actList">
   <property name="text">
    <string>Список напоминаний (Ctrl+L)</string>
   </property>
  </action>
  <action name="actHistList">
   <property name="text">
    <string>История напоминаний (Ctrl+H)</string>
   </property>
  </action>
  <action name="actAutoLoad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Автозагрузка</string>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="actAdd">
   <property name="text">
    <string>Добавить напоминание (Ctrl+F)</string>
   </property>
  </action>
  <action name="actNoSound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Отключить</string>
   </property>
  </action>
  <action name="actionbushorn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>bushorn</string>
   </property>
  </action>
  <action name="actioncow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>cow</string>
   </property>
  </action>
  <action name="actionhey">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>hey</string>
   </property>
  </action>
  <action name="actionkostel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>kostel</string>
   </property>
  </action>
  <action name="actionphone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>phone</string>
   </property>
  </action>
  <action name="actMySound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Свой звук...</string>
   </property>
  </action>
  <action name="actAddFiles">
   <property name="text">
    <string>Выбрать файлы...</string>
   </property>
  </action>
  <action name="actAddDirs">
   <property name="text">
    <string>Выбрать каталоги...</string>
   </property>
  </action>
  <action name="actCancelFiles">
   <property name="text">
    <string>Отменить выбор файлов</string>
   </property>
  </action>
  <action name="actCancelDirs">
   <property name="text">
    <string>Отменить выбор каталогов</string>
   </property>
  </action>
  <action name="actAutoOpenFiles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Открывать файлы автоматически</string>
   </property>
  </action>
  <action name="actAutoOpenDirs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Открывать каталоги автоматически</string>
   </property>
  </action>
  <action name="actionReadme_txt">
   <property name="text">
    <string>Readme.txt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrcrem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actHide</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
